providers/burstsms/src/lib/burstSms.provider.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  ChannelTypeEnum,
+  ISendMessageSuccessResponse,
+  ISmsOptions,
+  ISmsProvider,
+} from '@novu/stateless';
+import axios, { AxiosInstance } from 'axios';
+import qs from 'qs';
+
+export class BurstSmsSmsProvider implements ISmsProvider {
+  id = 'burstSms';
```

Comment:
```suggestion
  id = 'burstsms';
```
Is the value defined in the provider enum of the shared module.
Reference: libs/shared/src/consts/providers/provider.enum.ts